.github/workflows/ci.yml

```diff
@@ -23,9 +25,33 @@ jobs:
     - uses: actions/setup-java@v1
       with:
         java-version: 11
-    - run: |
-        mvn --batch-mode package
+    - run: mvn --batch-mode deploy
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    # will be uncommented as soon as sonar token is available
+    # - run: mvn --batch-mode verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
+    #   env:
+    #     SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+    #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - uses: actions/upload-artifact@v1
       with:
         name: target
         path: target
+  package:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/download-artifact@v1
+      with:
+        name: target
+        path: target
+    - run: sudo apt-get install --yes --no-install-recommends libxml-xpath-perl
+    - run: |
+        VERSION=$(xpath -q -e "/project/version/text()" pom.xml)
```

Comment:
Can be done using plain Maven:
```
mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate -Dexpression=project.version -q -DforceStdout
```

Reply:
yeah but on i think it takes longer - at least in bigger projects this was up to 90 seconds from my expirience